python/pyarrow/tests/test_substrait.py

```diff
@@ -22,6 +22,7 @@
 import pyarrow as pa
 from pyarrow.lib import tobytes
 from pyarrow.lib import ArrowInvalid
+from pyarrow.substrait import make_extension_id_registry
```

Comment:
nit: This should probably come after the `try-catch` block for checking non-erroneous substrait import.